lego/utils/models.py

```diff
@@ -21,7 +23,7 @@ def __init__(self, *args, **kwargs):
                 if isinstance(v, abakus_cached_property):
                     self._cached_properties[k] = v
 
-    def save(self, *args, **kwargs):
+    def save(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
There's no other way of typing this?

Reply:
Yes :)
It is a big job to fix, and I do agree that re-typing these is very tedious so if there is a way to tell mypy that the method should have the same signature as the parent class than that would solve a lot of problems.

I  couldn't find an easy way to use `*args, **kwargs` when overriding a typed method in mypy though so it might be more trouble than it's worth... Thats python typing for you :3rd_place_medal:.

For reference, here is the method in django: https://github.com/django/django/blob/04fdf719331dde6b0f4e7cdc445be4d4278f3ec4/django/db/models/base.py#L767-L769
and the type signature in django-stubs: https://github.com/typeddjango/django-stubs/blob/a7a15185a5ede9b487cf40ec02da08527b81afc5/django-stubs/db/models/base.pyi#L46-L52

Reply:
Yup, very tedious work, so I think I can leave that for later, or just wait for `mypy` to support inheritance 🙄 .